src/test/common/test_util.cc

```diff
@@ -16,11 +16,15 @@
 #include "include/util.h"
 #include "gtest/gtest.h"
 
-#include <sstream>
+#include <experimental/filesystem>
 
 #if defined(__linux__)
 TEST(util, collect_sys_info)
 {
+  if (!std::experimental::filesystem::exists("/etc/os-release")) {
+    GTEST_SKIP() << "skipping as '/etc/os-release' does not exist";
```

Comment:
we can also just *return* if `collect_sys_info()` returns false. but i think the better (and probably more correct) approach is to *skip* this test instead.

Reply:
agreed, it makes it clear when a test is/isn't actually run